getinsomnia/insomnia
# Fixes ApiSpec content syncing

closes INS-1393

To test this PR:
- type into the code editor in the design tab and observe that it updates the swagger-ui preview as normal
- import the example OpenAPI spec by clicking the "import from example"
- import an existing OpenAPI spec from a URL, for example https://gist.githubusercontent.com/gschier/4e2278d5a50b4bbf1110755d9b48a9f9/raw/801c05266ae102bcb9288ab92c60f52d45557425/petstore-spec.yaml
- import an existing OpenAPI spec from file (can use the above, if you like)
